.github/workflows/ci-deploy.yml

```diff
@@ -5,7 +5,8 @@ jobs:
     name: Build and Test Images
     runs-on: ubuntu-latest
     # NOTE (DP): Publish on develop and master, test on PRs against these
-    if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master' || github.base_ref ==  'develop' || github.base_ref ==  'master'
+    # TODO(DP) Reinstate CRONed release builds to update stock apps regularly
+    if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/develop-6.x.x' || github.base_ref ==  'develop' || github.base_ref ==  'develop-6.x.x'
```

Comment:
Why should we remove master from the list of branches to deploy from?

Reply:
I would say we can remove develop here _but not_ master.

Reply:
@line-o not really. For now `develop` deploys `latest`, and this here replaces `release` aka `master`. 

There is a question is if we plan further merges to `master` from `develop-6.x.x` or if the next merge will be from `develop` given the divergent histories of dev and dev6.

Reply:
One thing is for sure: this code will never be in `develop` 
So we can remove it from the condition.
Whether the code ends on master is a different question.
Just to keep that option I would want to see the condition to change to
```
if: github.ref == 'refs/heads/develop-6.x.x' || github.ref == 'refs/heads/master' || github.base_ref == 'develop-6.x.x' || github.base_ref == 'master'
```

Reply:
As discussed in tonights call we will have all three branches in here